docs/05-engineering/systems-sandboxes.md

```diff
@@ -1,9 +1,290 @@
 # Systems and Sandboxes
 
-> @todo [NEEDS WORK](https://trello.com/c/MerOhxSl)
-
 * We contemplate DevOps lite so that we influence greater harmony and integration between sandboxes and systems.
 
   ![DevOps Lite](../../images/devops-areas.png)
 
 * We document our systems fully while avoiding too much verbosity.
+
+## Table of Contents
```

Comment:
I'm shocked we don't have anytihng about Bowline in here... it would seem to replace most of this manual config I am seeing below. Maybe I misunderstood what thi sis